README.md

```diff
@@ -112,6 +112,18 @@ with lock:
 ```
 
 
+### Adjusting the Log Level
+
+By default, the logger name is *"filelock"* and the logging level is set to
+*logging.WARNING*. This can be adjusted using the *logging* module.
```

Comment:
This was INFO, but we changed it to a warning. Now all log messages are DEBUG AFAIK.

Reply:
@gaborbernat right, I meant to describe the logger's default level (since its not specified, it defaults to [`WARNING`](https://docs.python.org/3/howto/logging.html#when-to-use-logging)) rather than the level of the messages that the library emits.

Any suggestions on wordsmithing here to make that more clear?

Reply:
I don't think we need to document this, and your description actually is not true because things showing up depends on the level for both the logger and the log handler. IMHO we can just say that:

All log messages by this library are made using the DEBUG level. On how to control displaying/hiding that please consult the [logging documentation of the standard library](https://docs.python.org/3/howto/logging.html`). E.g. to hide these messages you can use `logging.getLogger("filelock").setLevel(logging.INFO)`.


We can't say anything about actually showing it because for that we'd need to know how the log handler is set up.

Reply:
That's a good point. See the updated changes